modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/DatePicker/DatePicker.es.js

```diff
@@ -78,19 +59,41 @@ const getDelimiter = (dateFormat) => {
 	return dateDelimiter;
 };
 
-const getDateFormat = () => {
-	const dateFormat = Liferay.AUI.getDateFormat();
-	const dateDelimiter = getDelimiter(dateFormat);
+const getLocaleDateFormat = (locale, format = 'L') => {
+	moment.locale(locale);
```

Comment:
@natocesarrego, I needed to put this line here to fix a bug. But in this case the chaining is not possible.